.github/workflows/CICD.yml

```diff
@@ -11,7 +11,7 @@ env:
   PROJECT_NAME: coreutils
   PROJECT_DESC: "Core universal (cross-platform) utilities"
   PROJECT_AUTH: "uutils"
-  RUST_MIN_SRV: "1.32.0" ## v1.32.0 - minimum version for half, tempfile, etc
+  RUST_MIN_SRV: "1.33.0" ## v1.33.0 - minimum version for libc required on aarch64-apple
```

Comment:
OK, could you please update the doc too https://github.com/uutils/coreutils/blob/master/README.md#rust-version ?

Reply:
half and tempfile are probably requiring something more recent than 1.33, no?
if so, could you please put back the comment